app/models/response/emcsTfe/GetMovementResponse.scala

```diff
@@ -30,6 +30,7 @@ case class GetMovementResponse(arc: String,
                                deliveryPlaceTrader: Option[TraderModel],
                                localReferenceNumber: String,
                                eadStatus: String,
+                               consignorTrader: ConsignorTraderModel,
```

Comment:
Can we use `TraderModel` here?

Reply:
I think in the other frontends it has been reused but here I felt it would better reflect the 801 schema where `consigneeTrader` is [optional](https://github.com/hmrc/emcs-tfe-stub/blob/main/conf/resources/schemas/ie801.xsd#L29) and `consignorTrader` is [mandatory](https://github.com/hmrc/emcs-tfe-stub/blob/main/conf/resources/schemas/ie801.xsd#L31) plus the difference of [Traderid](https://github.com/hmrc/emcs-tfe-stub/blob/main/conf/resources/schemas/ie801.xsd#L49-L65) vs [TraderExciseNumber](https://github.com/hmrc/emcs-tfe-stub/blob/main/conf/resources/schemas/ie801.xsd#L77-L92)